docs/workflow_solutions/using_anaconda.md

```diff
@@ -282,6 +282,94 @@ Now we can be sure that the correct versions of the software will be installed o
+
+    ```bash
+    conda search -c conda-forge numpy
+    ```
+
+If we find the package at one of these sources, we check the Platform version to ensure it is either noarch (if available) or linux. Noting the version, we can click the "source" or "repo" link (if available) or "homepage". Then we try to find the latest version. For a package found on GitHub, click "Releases" on the right-hand side. Verify that the latest Release is the same as, or very close to, the version on Anaconda or Pypi. If so, the package is being maintained on Anaconda/Pypi and suitable for use. Note the exact software name, version, and channel (if not on Pypi). ![!Github page for numpy, an Anaconda package](images/github_conda_releases.png)
+
+If we don't find it in any of the sources mentioned above, or the Anaconda/Pypi pages are out of date, then we may not be able to use the software in an Anaconda environment. It is possible to try installing a git repository using pip, but care must be taken to choose the right commit or tag, you find more [info](https://pip.pypa.io/en/stable/cli/pip_install/#examples). To search for a git repository try:
+
+1. github "name".
+2. gitlab "name".
+
+Remember to replace name with name of Anaconda package.
+
+When we have a complete list of Anaconda packages and Channels, then we can create an environment from scratch with all the dependencies included. For Anaconda packages, add one line to dependencies for each software. For Pypi packages add - pip: under dependencies. Then under - pip:add `==` to pin the version, see below.
+
+```yaml
+name: test-env
+channels:
+  - anaconda
+  - conda-forge
+  - pip
+dependencies:
+  - matplotlib=3.8.4 #pinned version for conda
+  - python=3.10.4
+  - pip
+  - pip:
+  - numpy==1.26.4 #pinned version for pip
+  - http://insert_package_link_here #for url links
+
+```
+
+ For git repos, add them under `- pip:` based on examples [here](https://pip.pypa.io/en/stable/cli/pip_install/#examples). See the section [Replicability versus Portability](#replicability-versus-portability) for more information.
+
+The above configuration is only for illustration purposes, to show how channels and dependencies can be used. It is best to install all of your packages from conda channels, to avoid your environment breaking. Only packages that are unavailable via conda, should be installed via pip. If you run into challenges please [contact us](../index.md#contact-us).
+
+##### Key Things To Remember
+
+1. Exploring Package Documentation: For each package, check the documentation to understand its features, version history, and compatibility. Documentation can often be found on the Anaconda Cloud package page under the "Documentation" or "Homepage" link shared above in this tutorial.
+
+1. Regularly update your environment file to manage dependencies and maintain compatible software environments.
```

Comment:
Regularly _consider_ updating...

We want people to consider it, not do it blindly. Updating in the middle of an experiment could be an unhelpful choice. It could also break their workflows if major versions change and aren't backward compatible with their code.

We may want to also give the reason that "newer software tends to resolve older bugs, improving the state of science".